index.bs

```diff
@@ -2128,7 +2171,7 @@ Also, to prevent CORS-related vulnerabilities each page will see a new instance
 Fingerprinting {#fingerprinting-security}
 --------------
 
-Given that the API describes hardware available to the user and its capabilities it will inevitably provide additional surface area for fingerprinting. While it's impossible to completely avoid this, steps can be taken to mitigate the issue. This spec limits reporting of available hardware to only a single device at a time, which prevents using the rare cases of multiple headsets being connected as a fingerprinting signal. Also, the devices that are reported have no string identifiers and expose very little information about the devices capabilities until an XRSession is created, which may only be triggered via user activation in the most sensitive case.
+Given that the API describes hardware available to the user and its capabilities it will inevitably provide additional surface area for fingerprinting. While it's impossible to completely avoid this, user agents should take steps to mitigate the issue. This spec limits reporting of available hardware to only a single device at a time, which prevents using the rare cases of multiple headsets being connected as a fingerprinting signal. Also, the devices that are reported have no string identifiers and expose very little information about the devices capabilities until an XRSession is created, which requires additional protections when [=sensitive information=] will be exposed.
```

Comment:
There are explicit requirements for user agents to anonymize values which may be unique to the device. Do you want to call out those scenarios here? For reference, those requirements apply to:

- XRPose data, when based on underlying raw sensor data (to avoid fingerprinting sensor calibration data)
- Factory-configured data including IPD or screen geometry data
- User-configured data including IPD, viewer height, or display locations for CAVE systems

Reply:
Also, sensitive data that could be used for user profiling does not appear to be addressed. Will that be in a different PR?